frontend/apps/reader/modules/readerview.lua

```diff
@@ -51,10 +51,6 @@ local ReaderView = OverlapGroup:extend{
     scroll_mode = "vertical",
     -- properties of the gap drawn between each page in scroll mode:
     page_gap = {
-        -- width in pixels (when scrolling horizontally)
-        width = Screen:scaleBySize(G_reader_settings:readSetting("page_gap_width") or 8),
```

Comment:
if you are getting rid of this you should also clean up
https://github.com/koreader/koreader/blob/7fb2db8f02897b8ff27f529f752f9a6150ca9939/frontend/apps/reader/modules/readerview.lua#L392-L396
and
https://github.com/koreader/koreader/blob/7fb2db8f02897b8ff27f529f752f9a6150ca9939/frontend/apps/reader/modules/readerview.lua#L51

does anyone know what `scroll_mode` is used for and if it is ever set to `horizontal`?

Reply:
I renamed scroll mode to continuous in the GUI.

Reply:
@Frenzie 
`scroll_mode` or `page_scroll`?

Reply:
`page_scroll` is a boolean that holds `True` in continuous mode, `False` in page mode. `scroll_mode` set `vertical` in `ReaderView` creation and doesn't change.

Reply:
That's part of the same thing, perhaps separated specifically for that vertical vs horizontal thing. I don't know the specific plan behind 63e5e7e577a0fb18b27f0b50400e38ef650b3a71